cmd/resource_cmds_test.go

```diff
@@ -0,0 +1,42 @@
+package cmd_test
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+
+	"ldcli/cmd"
+	"ldcli/internal/analytics"
+)
+
+func TestCreateTeam(t *testing.T) {
+	t.Run("help shows postTeam description", func(t *testing.T) {
+		args := []string{
+			"teams",
+			"create",
+			"--help",
+		}
+
+		output, err := cmd.CallCmd(t, cmd.APIClients{}, &analytics.NoopClient{}, args)
+
+		require.NoError(t, err)
+		assert.Contains(t, string(output), "Create a team.")
+	})
+	t.Run("with valid flags calls makeRequest function", func(t *testing.T) {
+		args := []string{
+			"teams",
+			"create",
+			"--access-token",
+			"abcd1234",
+			"--data",
+			`{"key": "team-key", "name": "Team Name"}`,
+		}
+
+		output, err := cmd.CallCmd(t, cmd.APIClients{}, &analytics.NoopClient{}, args)
+
+		require.NoError(t, err)
+		s := string(output)
+		assert.Contains(t, s, "would be making a request to /api/v2/teams here, with args: map[data:map[key:team-key name:Team Name] expand:]\n")
```

Comment:
haven't mocked out a client yet since we're not actually making any requests. will update this test with a mock client when that's a thing